core/services/workflow_service/controllers/project_controller.py

```diff
@@ -0,0 +1,205 @@
+from uuid import UUID, uuid4
+from fastapi import HTTPException
+from sqlalchemy.orm import Session
+import datetime
+from typing import List
+
+from utils.database.session_injector import get_database
+from core.services.workflow_service.models.project import Project
+from services.user_service.models.user import User
+from core.services.workflow_service.models.block import Block
+
+
+# Does current uuid need to be extracted from the token?
+def create_project(name: str,  current_user_uuid: UUID) -> UUID:
+    db: Session = next(get_database())
+    project: Project = Project()
+
+    project.uuid = uuid4()
+    project.name = name
+    project.created_at = datetime.utcnow()
+    current_user = (db.query(User).filter_by(uuid=current_user_uuid)
+                    .one_or_none())
+
+    if not current_user:
+        raise HTTPException(404, detail="User not found")
+
+    # TODO: add relation with blocks
+
+    db.add(project)
+    db.commit()
+    db.refresh(project)
+
+    return project.uuid
+
+
+def read_project(project_uuid: UUID) -> Project:
+    db: Session = next(get_database())
+
+    project = db.query(Project).filter_by(uuid=project_uuid).one_or_none()
+
+    if not project:
+        raise ValueError("Project not found")
```

Comment:
Use `HTTPException`. Please check all occurences of `ValueError`. I think this makes sense in all of the usages (especially within the controller)